### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by bcakmakoglu** [bcakmakoglu/vue-flow#146] [Announcements]
For anyone wanting to join the Vue Flow discord server you can use this [invitation](https://discord.gg/rwt6CBk4b5).

It's meant for people to exchange about issues or solutions regarding Vue Flow or just in general ask questions.
For anyone trying to contact me, it's probably also easier to get a hold of me there than via mail or something 😄

**Comment by grinsli** [comment]
Hello hello. Could you update the invite please, doesn't seem to work anymore. 
If the server is still valid that is.

**Reply by bcakmakoglu** [reply]
Sure, link is updated!

**Comment by niemes** [comment]
The discord link doesn't work anymore.
